sdk/devcenter/ci.yml

```diff
@@ -22,12 +20,14 @@ pr:
     - restapi*
   paths:
     include:
-    - sdk/devcenter/
+    - sdk/devcenter/azure-developer-devcenter/
+    - sdk/devcenter/ci.yml
 
 extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: devcenter
++   TestProxy: true
     Artifacts:
-    - name: azure-mgmt-devcenter
-      safeName: azuremgmtdevcenter
+    - name: azure-developer-devcenter
+      safeName: azuredeveloperdevcenter
```

Comment:
@chrissmiller Please only add 
`    - name: azure-developer-devcenter
      safeName: azuredeveloperdevcenter` 
after `azure-mgmt-devcenter` like: https://github.com/Azure/azure-sdk-for-python/blob/0e104a6c129c6c59309d4bfbe2d525dc1b71c1a7/sdk/webpubsub/ci.yml#L32-L35. You can't change content about mgmt-devcenter otherwise mgmt release will be blocked

Reply:
Ah thanks! Thought I had to use the ci.mgmt.yml convention from other repos. Fixed!